site/docs/pages/token/introduction.mdx

```diff
@@ -15,8 +15,10 @@ TokenKit enables you to effortlessly create user experiences using Token compone
 The available components are:
 
 - [`<TokenChip />`](/token/token-chip): Small button component for a given token.
+- [`<TokenImage />`](/token/token-image): Image component for token image.
 - [`<TokenRow />`](/token/token-row): Row component for a given token.
 - [`<TokenSearch />`](/token/token-search): Search component to search by name, symbol and address for a given list of tokens.
+- [`<TokenSelector />`](/token/token-selector): Stylized button component to display token info or placeholder text.
```

Comment:
Can TokenChip maybe have a selector option 🤔, that's a very temptive idea, but not sure if will make things more complex.